# socket_communication/msg/Opencv1.msg
int32 data_flag
int32 flag_condition